src/View/Requirements_Backend.php

```diff
@@ -494,9 +494,47 @@ protected function getAllJavascript()
     public function customScript($script, $uniquenessID = null)
     {
         if ($uniquenessID) {
-            $this->customScript[$uniquenessID] = $script;
+            $this->customScript[$uniquenessID]['content'] = $script;
```

Comment:
$this->customScript has protected visibility instead of private so it's technically API which we shouldn't change until the next major version (CMS 6)

Instead of add a 'content' key, you could create a new `private array $customScriptAttributes = [];` property to hold the custom attributes

To get the index to add to when adding scripts without a $uniquenessID do something like `$index = max(array_map('intval', array_keys($this->customScript))) + 1;` and then $this->customScriptAttributes[$index] = $attributes;

Reply:
Sure thing, I'll have this changed in my next commits.

Reply:
Looks like this one still wasn't updated. Do either of you have thoughts about this?

Reply:
Oh good catch.  Yeah it should just be reverted to `$this->customScript[$uniquenessID] = $script;`